### ISSUE TITLE
require 'webmock/rspec' fails

### ISSUE THREAD

**Issue by marcandre** [deep-cover/deep-cover#7]
Haven't investigated, but:

```
$ gem install 'webmock'
$ ruby -e "require 'webmock/rspec'; puts 'ok'"  # => ok
$ ruby -e "require 'deep_cover/builtin_takeover'; Coverage.start; require 'webmock/rspec'"
# => gems/deep-cover-0.1.3/lib/deep_cover/core_ext/require_overrides.rb:9:in `require': cannot load such file -- webmock/rspec (LoadError)
```

@maxlap can you check it out?

**Comment by marcandre** [comment]
Mmm, I somehow need to call `gem 'webmock'` before the `require` to avoid this. We should do it automatically...

**Comment by MaxLap** [comment]
This is fixed in 0.1.7
